WordPress/Classes/Utility/WPWebViewController.h

```diff
@@ -42,6 +42,11 @@
 @property (nonatomic, assign) BOOL      secureInteraction;
 
 /**
+ *  @brief  When true adds a custom referrer to NSURLRequest.
+ */
+@property (nonatomic, assign) BOOL useCustomReferrer;
```

Comment:
What do you think of `appendsWPComReferrer`? Whereas `useCustomReferrer` might imply that a referrer could be specified. Just nitpicking.